torch/__init__.py

```diff
@@ -1869,5 +1869,59 @@ def __getattr__(name):
 
         raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
 
+
+def _constrain_as_value(symbol, min: Optional[builtins.int] = None, max: Optional[builtins.int] = None):
```

Comment:
IMO this is a useless indirection. Why not replace throughout by `torch.sym_constrain_range[_for_size]`?